src/StatusQ/Controls/StatusIdenticonRing.qml

```diff
@@ -0,0 +1,56 @@
+import QtQuick 2.0
+import QtQuick.Layouts 1.0
+
+Item {
+    id: identiconRing
+
+    property var ringSpec: undefined // Expected ListModel object to iterate through ring segments
+    property real totalRingUnits: 1
+    property real initalAngleRad: 0
+    property real ringPxSize: 1.5
+
+    readonly property real segmentRadLen: 2 * Math.PI / identiconRing.totalRingUnits
+    // 32 distinctive segment: They are expected to be used using the array index from 0 to 31 and with the order defined in figma (it should be defined as well in specs.)
+    readonly property var distinctiveColors: ["#000000", "#726F6F", "#C4C4C4", "#E7E7E7", "#FFFFFF", "#00FF00",
+                                              "#009800", "#B8FFBB", "#FFC413", "#9F5947", "#FFFF00", "#A8AC00",
+                                              "#FFFFB0", "#FF5733", "#FF0000", "#9A0000", "#FF9D9D", "#FF0099",
+                                              "#C80078", "#FF00FF", "#900090", "#FFB0FF", "#9E00FF", "#0000FF",
+                                              "#000086", "#9B81FF", "#3FAEF9", "#9A6600", "#00FFFF", "#008694",
+                                              "#C2FFFF", "#00F0B6"]
```

Comment:
Yes, it is something we should ask @John-44 because as far as I know, colored ring should be unique so don't know if it could be changed, in the perspective of identification, depending on the theme. But surely, these colores are not correct for both views, so yes, it makes more sense to have a pair for each.

Reply:
@noeliaSD the colours should be the exactly the same across themes e.g. the ring colours shouldn't change when the user changes themes.

Reply:
> Oh and they should be added to the design system figma then. We already have a color palette system there. Could introduce an additional section for "verified" colors.

I think @davholik is working on introducing these colours into the design system file. We can keep it now just as it is and when he finishes, we can create a new task for doing the refactor! WDYT?

Reply:
I just discussed this with @John-44, ideally, we want to try to mirror the identicon color palette in the dark theme with slightly tweaked properties, so it works in a dark environment, but still keeps the unique properties. We need to experiment with this.

So what I would do right now is, that prepare the code in a way that can support separate dark theme variant of this palette. For now, the dark variation of the palette will be identical to the light one but it will be prepared for any future changes.

I am including this identicon palette in the new branch of the design system. It is definitely gonna sit in it's separate group. It will be either part of the data visualization color group or its own. Bring it into the code as a separate group for sure.

Reply:
@John-44 @davholik 

In order to have all client apps aligned, these colors should come from status-go. Otherwise, we will end up either with different identicon ring colors on mobile and desktop or with colors definition duplication in qml and react, which will be hard to maintain and bug-prone. If we really want to tweak the properties by theme it should be done by appropriate client's transform function, e.g. Qt.darker(identiconRingColors[0], 1.15).

Reply:
> If we really want to tweak the properties by theme it should be done by appropriate client's transform function, e.g. Qt.darker(identiconRingColors[0], 1.15).

So the thing with this is that it really depends on what's the impact of tweaks needed to have those colors work in different themes. Lemme elaborate:

1. Often times, a color in one theme can't be mapped to another color in a different theme using calculated rules (.e.g `Qt.darker(...)`). In those cases, the color `red` in light mode isn't just the same `red` in a different shade in darkmode, it's actually a *different* `red`. We have a good amount of such cases in our color palette which is why our `Theme` system is so explicit. 
2. I think there's plans to eventually support custom themes, so users will be able to install themes for Status Desktop, in which case the colors provided by status-go might not go well with the theme at all.

I think we have a handful of options with their trade-offs:

1. We make color rings look exactly the same regardless of what the theme looks like (proposed by @John-44), at the cost of it possibly not being pleasant to the eye in any other theme that wasn't designed with those colors in mind (even in darkmode already those colors look too sharp, which is probably why @davholik wants to tweak them on a per theme basis).
2. We let the theme decide what those ring colors are, ensuring all colors work well in every theme, however, at the cost of the rings to be hard to recognize/verify across themes (as they might look different)
3. We drop the idea of supporting custom themes in the future entirely, at which point we'll only need to support light and darkmode colors which can then be either defined in our themes on the client side, or come from status-go (doesn't really matter)

The argument that this can cause inconsistencies across apps (desktop, mobile, etc..) is a good one. Which probably means we'll have to go with 1) or 3)

Reply:
> 1. Often times, a color in one theme can't be mapped to another color in a different theme using calculated rules (.e.g `Qt.darker(...)`).

Yes, that's obvious, but it is not what I meant. I put it in the context, what I wanted to state is:
All colors should come from status-go (and so be the same for every theme) **ONLY THEN**
>If we really want to tweak the properties by theme it should be done by appropriate client's transform function, e.g. Qt.darker(identiconRingColors[0], 1.15).

Reply:
> 1. We make color rings look exactly the same regardless of what the theme looks like (proposed by @John-44), at the cost of it possibly not being pleasant to the eye in any other theme that wasn't designed with those colors in mind (even in darkmode already those colors look too sharp, which is probably why @davholik wants to tweak them on a per theme basis).

I opt for this one. Identicon ring is a mechanism for visual Identity representation, it is meant to be consumed subconsciously by the Users. In my opinion, if colors will differ per Theme human brain will not handle it.

Reply:
Taking into account all your comments and since we still have some solutions open, I suggest the following:
- To make `StatusIdenticonRing` the most flexible _StatusQ_ component, we can expose the `distinctiveColors` property as a required property to be set by the consumer. Then we can close the component itself as it is.
- To manage colors/themes, we can now add the list of colors (all 32 defined in figma) in the Themes (now the same for light and dark) and when we use the component in the demo app, just like we are configuring the model in case of _ringSpecsModel property_, we can set up an array of the 32 colors (using Themes) we've now defined. So, we will have prepared the definition of colors in different themes (just waiting for the dark mode updates) and we will leave the door open to give the responsibility to the backend to decide which colors are passed to the ring.

WDYT?

Reply:
Sounds like a good "best-of-both-worlds" solution - I like it